src/test/scala/tech/sourced/gemini/BaseSparkSpec.scala

```diff
@@ -2,32 +2,50 @@ package tech.sourced.gemini
 
 import org.apache.spark.SparkConf
 import org.apache.spark.sql.SparkSession
-import org.scalatest.{BeforeAndAfterAll, Suite}
+import org.scalatest.{BeforeAndAfterAll, Suite, Tag}
 
 trait BaseSparkSpec extends BeforeAndAfterAll {
   this: Suite =>
 
-  @transient var sparkSession: SparkSession = _
+  var sparkSession: SparkSession = _
```

Comment:
Could you please remind, what is the reason to change it here?

Reply:
the only reason is to simplify the code.
I couldn't find why we need to set spark session unserializable here, so looks like we don't need.
To avoid wasting the time of the next developer who stumble onto it I removed it.

Reply:
I see. I would rather keep it transient, as it follow the same pattern as tests in [Apache Spark do](https://github.com/apache/spark/blob/ceaf77ae43a14e993ac6d1ff34b50256eacd6abb/mllib/src/test/scala/org/apache/spark/mllib/util/MLlibTestSparkContext.scala#L30) - it seems have to do something with ability to make sure that there is only one SparkContext in JVM.

Not sure though, how that could happen in our case, but at least for the sake of [consistency](https://github.com/holdenk/spark-testing-base/blob/c6b4c74ea6274b2a13c9226dbdb82f435387b60d/src/main/2.0/scala/com/holdenkarau/spark/testing/DataFrameSuiteBase.scala#L57) I would keep it as is.

Reply:
reverted. But I don't like having magic copy-past without understanding why we need it.